### PR TITLE
Removing unnecessary "Table of" from table caption strings.

### PR DESCRIPTION
**Pull request by crystalenka**
Pull Request for Issue #35279 .

### Summary of Changes

Screen readers announce the element type by default; if an individual has set their device to not announce element types, that is their choice. Similar to having "image of" in alt text or "link to" in a link.

See https://russmaxdesign.github.io/accessible-forms/accessible-name-table04.html for how different screen readers handle table captions by default.

### Testing Instructions

With your favorite screenreader, navigate to any core component table view and listens how it announces the table.

#### VoiceOver
Chrome: Table of [component]. Table. 2 Columns. 3 rows.
Firefox: Table of [component]. Table. 2 Columns. 3 rows.
Safari: Table of [component]. Table. 2 Columns. 3 rows.

#### NVDA
Chrome: Table with 3 rows and 2 columns. Table of [component].
Firefox: Table with 3 rows and 2 columns. Table of [component].
Safari: Table with 3 rows and 2 columns. Table of [component].

#### JAWS
Chrome: Table with 2 columns and 3 rows. [Name not announced]
Firefox: Table with 2 columns and 3 rows. [Name not announced]
Safari: Table with 2 columns and 3 rows. [Name not announced]

#### VoiceOver
Chrome: [component]. Table. 2 Columns. 3 rows.
Firefox: [component]. Table. 2 Columns. 3 rows.
Safari: [component]. Table. 2 Columns. 3 rows.

#### NVDA
Chrome: Table with 3 rows and 2 columns. [component].
Firefox: Table with 3 rows and 2 columns. [component].
Safari: Table with 3 rows and 2 columns. [component].

#### JAWS
Chrome: Table with 2 columns and 3 rows. [Name not announced]
Firefox: Table with 2 columns and 3 rows. [Name not announced]
Safari: Table with 2 columns and 3 rows. [Name not announced]

### Documentation Changes Required

None that I'm aware of. Perhaps in developer documentation regarding accurate table captions

### CODE REVIEW

**Comment by Kostelano** [comment]
Missed. Please make corrections.

administrator\language\en-GB\com_contenthistory.ini
`COM_CONTENTHISTORY_COMPARE_CAPTION="Table comparing two versions."`

administrator\language\en-GB\com_scheduler.ini
`COM_SCHEDULER_TABLE_CAPTION="Tasks List"`

language\en-GB\com_contact.ini
`COM_CONTACT_TABLE_CAPTION="Table of Contacts"`

language\en-GB\com_content.ini
`COM_CONTENT_ARTICLES_TABLE_CAPTION="Table of Articles"`
`COM_CONTENT_CATEGORY_LIST_TABLE_CAPTION="List of articles in category %s"`

language\en-GB\joomla.ini
`JGLOBAL_REPEATABLE_FIELDS_TABLE_CAPTION="Table of repeatable fields."`

**Comment by crystalenka** [comment]
Thanks @Kostelano , added those and a couple others I missed originally. Nice catch!

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on 7c4e32fca94d596ab711dbc17123242d5d4b0c25<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on caddbf8860963c0fe1f1e5e53aef8b8d7fe09b84<br /><br />Because of the latest commit.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on caddbf8860963c0fe1f1e5e53aef8b8d7fe09b84<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 1]
```diff
@@ -48,4 +48,4 @@ PLG_SYSTEM_WEBAUTHN_MANAGE_HEADER_NOMETHODS_LABEL="No authenticators have been s
 PLG_SYSTEM_WEBAUTHN_MSG_DELETED="The authenticator has been removed."
 PLG_SYSTEM_WEBAUTHN_MSG_SAVED_LABEL="The label has been saved."
 PLG_SYSTEM_WEBAUTHN_REQUIRES_GMP="Either of the PHP extensions GMP or BCmath must be loaded to add authenticators."
-PLG_SYSTEM_WEBAUTHN_TABLE_CAPTION="Table of WebAuthn authenticators."
+PLG_SYSTEM_WEBAUTHN_TABLE_CAPTION="WebAuthn Authenticators."
```
Comment (Quy) [inline administrator/language/en-GB/plg_system_webauthn.ini]:
```suggestion
PLG_SYSTEM_WEBAUTHN_TABLE_CAPTION="WebAuthn Authenticators"
```

File: administrator/language/en-GB/com_languages.ini [hunk 2]
```diff
@@ -69,7 +69,7 @@ COM_LANGUAGES_MULTILANGSTATUS_CONTENT_LANGUAGE_MISSING="The <strong>%s</strong>
 COM_LANGUAGES_MULTILANGSTATUS_CONTENT_LANGUAGE_PUBLISHED="Published Content Languages"
 COM_LANGUAGES_MULTILANGSTATUS_CONTENT_LANGUAGE_SEF_MISSING="The \"URL Language Code\" is missing for the <strong>%s</strong> Content Language. <strong>The multilingual feature may not function properly or the site may not be reachable!</strong>"
 COM_LANGUAGES_MULTILANGSTATUS_CONTENT_LANGUAGE_TRASHED="The <strong>%s</strong> Content Language is trashed. <strong>The language will not display on the site.</strong>"
-COM_LANGUAGES_MULTILANGSTATUS_CONTENT_TABLE_CAPTION="Table of the status for each content language."
+COM_LANGUAGES_MULTILANGSTATUS_CONTENT_TABLE_CAPTION="The status of each content language."
```
Comment (Quy) [inline administrator/language/en-GB/com_languages.ini]:
```suggestion
COM_LANGUAGES_MULTILANGSTATUS_CONTENT_TABLE_CAPTION="Content Language Status"
```

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by brianteeman** [comment]
Sorry this is not correct 
api/joomla.ini is supposed to be in sync with the main joomla.ini

**Comment by richard67** [comment]
Back to pending. @crystalenka Could you check @brianteeman 's comment above and adjust the api/joomla.ini file, too? Thanks in advance.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by crystalenka** [comment]
There are other differences between api/joomla.ini and the main joomla.ini. Is there a standard that determines which items carry over and which ones do not?

**Comment by richard67** [comment]
> There are other differences between api/joomla.ini and the main joomla.ini. Is there a standard that determines which items carry over and which ones do not?

@crystalenka Which "main joomla.ini" did you check? As far as I know - but I might be wrong - the api file should be equal to the admin file.

**Comment by brianteeman** [comment]
@richard67 that is correct

**Comment by crystalenka** [comment]
@brianteeman @richard67 I think there was only one in the main joomla.ini. If I missed any other ones please let me know, but I think it's ready now.

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on f38dbe294e784c55067091ee67e68d888b22ced2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on f38dbe294e784c55067091ee67e68d888b22ced2<br /><br />Code Review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38761">issues.joomla.org/tracker/joomla-cms/38761</a>.</sub>

**Comment by roland-d** [comment]
Thank you.

**Pull request merged**